### ISSUE TITLE
ws version bump due to RangeError: out of range index crashing server

### ISSUE THREAD

**Issue by coffenbacher** [mikemintz/rethinkdb-websocket-server#26]
Is there much involved in bumping the ws version to >1.1.1? For now I manually bumped the version in my projects and it's working fine.

I've been seeing this bug a lot: https://github.com/websockets/ws/issues/778

**Comment by mikemintz** [comment]
We can do that, but we should make sure it doesn't break anything. Do you know what changed from 0.8.0 to 1.1.1? I'm looking at the [github releases](https://github.com/websockets/ws/releases) page and it talks about dropping support for very old versions of node, which is fine, but I can't see exactly what warranted the jump from 0.8 to 1.0.

**Comment by coffenbacher** [comment]
Yeah, breaking changes is why I didn't just go ahead with a pull request and opened an issue instead, I have no idea what's involved in making sure nothing else breaks.

At least in my team's personal experience with 2 small-medium projects: 1.1.1 works great, 0.8.0 gets hit hard with the websockets/ws#778 bug. So there's that "test in production" evidence but that's all we know.

**Comment by mikemintz** [comment]
That sounds promising. Can you go ahead and make a PR? Then I'll merge and tag a new version of rethinkdb-websocket-server at 0.7.0. At least if there are any issues for other projects, people won't be automatically updated unless they have a permissive dependency version range.
